services/frontend/src/scenes/Projects/Projects.scss

```diff
@@ -0,0 +1,115 @@
+@import "styles/_colors.scss";
+@import "styles/_dimens.scss";
+@import "styles/_mixins.scss";
+
+.projects {
+    display: flex;
+    flex-direction: column;
+    height: calc(100% - #{$navbar-height});
+    overflow: auto;
+}
+
+.projects-header {
+    border-bottom: 1px solid $color-text-secondary;
+    display: flex;
+    flex-shrink: 0;
+    justify-content: center;
+    padding: $default-spacing;
+}
+
+.filter-button {
+    background-color: transparent;
+    border: none;
+    border-radius: $default-border-radius;
+    box-shadow: none;
+    color: $color-text-secondary;
+    margin: 0 16px;
+    padding: $default-spacing/2 $default-spacing*1.5;
+    text-transform: none;
+
+    &.filter-button--active {
+        background-color: $color-primary;
+        color: $color-light;
+    }
+}
+
+.projects-list {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+    padding: $default-spacing * 2;
+
+    .project-card {
+        margin: $default-spacing 0;
+    }
+}
+
+.project-card {
+    display: flex;
+    flex-shrink: 0;
+    max-width: $default-spacing * 34;
+
+    .project-card-content {
+        display: flex;
+        padding: $default-spacing*1.5 $default-spacing*2;
+
+        .project-card-active-section {
```

Comment:
Maybe change to `project-card-activity-section` as it better implies this section holds the possible states (active _or_ inactive)

Reply:
Agreed, changed.